package.json

```diff
@@ -100,7 +103,7 @@
     "wait-on": "5.0.0",
     "webpack": "4.43.0",
     "webpack-cli": "3.3.11",
-    "webpack-dev-server": "3.10.3"
+    "webpack-dev-server": "3.11.0"
```

Comment:
I see @omarefg  ask about something similar.
https://github.com/Sphinx-Society/kaizen-web-client/pull/71#discussion_r426554076

And although it is accepted to increase the version, in fact because it was already done. Actually, the change is necessary because it fixes a bug. (in v3.10.3 npm audit fix -> v3.11.0)